src/csharp/Microsoft.Spark.Worker/Microsoft.Spark.Worker.csproj

```diff
@@ -8,6 +8,10 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="$(TargetFramework.StartsWith('net4'))">
+    <DefineConstants>$(DefineConstants);FEATURE_ASSEMBLY_LOADFROM</DefineConstants>
```

Comment:
You don't need to define your own constant for this. There are built-in constants generated for the TargetFramework that is currently being compiled.

In your C# code you can either say:

```C#
#if NETFRAMEWORK

// code only compiles for any .NET Framework version (net461, net45, etc)

#endif
```

or 

```C#
#if NET461

// code only compiles for net461

#endif
```

Reply:
Thanks!